doc/source/schemas/alleleAnnotations.rst

```diff
@@ -0,0 +1,562 @@
+  :field software:
+    The software run to generate this analysis.
+  :type software: array<string>
+  :field info:
+    A map of additional information.
+  :type info: map<array<string>>
+
+  An analysis contains an interpretation of one or several experiments.
+    (e.g. SNVs, copy number variations, methylation status) together with
+    information about the methodology used.
+    TODO: review
+
+.. avro:record:: Dataset
+
+  :field id:
+    The dataset's id, locally unique to the server instance.
+  :type id: string
+  :field name:
+    The name of the dataset.
+  :type name: null|string
+  :field description:
+    Additional, human-readable information on the dataset.
+  :type description: null|string
+
+  A Dataset is a collection of related data of multiple types.
+  Data providers decide how to group data into datasets.
+  See [Metadata API](../api/metadata.html) for a more detailed discussion.
+
+.. avro:record:: VariantSetMetadata
+
+  :field key:
+    The top-level key.
+  :type key: string
+  :field value:
+    The value field for simple metadata.
+  :type value: string
+  :field id:
+    User-provided ID field, not enforced by this API.
+      Two or more pieces of structured metadata with identical
+      id and key fields are considered equivalent.
+      `FIXME: If it's not enforced, then why can't it be null?`
+  :type id: string
+  :field type:
+    The type of data.
+  :type type: string
+  :field number:
+    The number of values that can be included in a field described by this
+      metadata.
+  :type number: string
+  :field description:
+    A textual description of this metadata.
+  :type description: string
+  :field info:
+    Remaining structured metadata key-value pairs.
+  :type info: map<array<string>>
+
+  Optional metadata associated with a variant set.
+
+.. avro:record:: VariantSet
+
+  :field id:
+    The variant set ID.
+  :type id: string
+  :field name:
+    The variant set name.
+  :type name: null|string
+  :field datasetId:
+    The ID of the dataset this variant set belongs to.
+  :type datasetId: string
+  :field referenceSetId:
+    The ID of the reference set that describes the sequences used by the variants in this set.
+  :type referenceSetId: string
+  :field metadata:
+    Optional metadata associated with this variant set.
+      This array can be used to store information about the variant set, such as information found
+      in VCF header fields, that isn't already available in first class fields such as "name".
+  :type metadata: array<VariantSetMetadata>
+
+  A VariantSet is a collection of variants and variant calls intended to be analyzed together.
+
+.. avro:record:: CallSet
+
+  :field id:
+    The call set ID.
+  :type id: string
+  :field name:
+    The call set name.
+  :type name: null|string
+  :field sampleId:
+    The sample this call set's data was generated from.
+      Note: the current API does not have a rigorous definition of sample. Therefore, this
```

Comment:
It is shouldn't read ```Sample``` but ```BioSample```, and this is defined both in PR https://github.com/ga4gh/schemas/pull/397 (in a basic version) as well as in the metadata branch https://github.com/ga4gh/schemas/blob/metadata/src/main/resources/avro/metadata.avdl#L441.

It would really be helpful if somebody from reads/ref/var would take care of merging 397 into master; it is basically needed by enough objects now, right?